ui/src/main/webapp/src/app/registered-application/application-list.component.ts

```diff
@@ -116,8 +116,8 @@ export class ApplicationListComponent extends ExecutionsMonitoringComponent impl
             return false;
 
         this.deleteAppDialog.data = application;
-        this.deleteAppDialog.title = 'Confirm application deletion';
-        this.deleteAppDialog.body = `Do you really want to delete the “${application.title}” application?`;
+        this.deleteAppDialog.title = 'Confirm Application Deletion';
+        this.deleteAppDialog.body = `Are you sure you want to delete “${application.title}”?`;
```

Comment:
here in dialogs are used the same quotes https://github.com/windup/windup-web/pull/403/commits/7220813aa413b63c2950cd7883518ddf631f2594#diff-d62f56f35f07760d38cacfbb5dd6860cR112 
I think this change is out of this task/US. I haven't added them, but I don't want to decide now for it.